cranelift/isle/isle/src/error.rs

```diff
@@ -19,13 +19,13 @@ impl std::fmt::Debug for Errors {
         }
         let diagnostics = Vec::from_iter(self.errors.iter().map(|e| {
             let message = match e {
-                Error::IoError { context, .. } => format!("{}", context),
+                Error::IoError { context, .. } => context.to_owned(),
```

Comment:
I think I'd prefer for this to call `clone`, or possibly `to_string`, rather than `to_owned`.

Reply:
Thats fair -- in my code, I usally bias towards `to_owned` or `clone` since its common for folks to refer to a string as either &str/str or String.

Changed it to clone!